dev/release/README.md

```diff
@@ -128,7 +128,7 @@ release.
 
 See [#9697](https://github.com/apache/datafusion/pull/9697) for an example.
 
-Here are the commands that could be used to prepare the `5.1.0` release:
+Here are the commands that could be used to prepare the `38.0.0` release:
```

Comment:
We need to get rid of hardcoded versions somehow....

Reply:
I should really try and automate more of the release process so that we just run scripts and pass in the version number